content-services/latest/admin/auth-sync.md

```diff
@@ -1,1823 +1,1864 @@
+
+* Configure the [authentication chain](#authchain) to the only supported value for the Identity Service: `authentication.chain=identity-service1:identity-service,alfrescoNtlm1:alfrescoNtlm`
+
+    >**Important:** The authentication chain can't contain any other values, such as Kerberos or SAML, when using the Identity Service.
+
+* Configure the [`alfresco-global.properties` file](#isprops)
+
+> **Note:** See the [supported platforms]({% link content-services/latest/support/index.md %}) page for the compatibility between Content Services and Identity Service.
+
+#### Identity Service configuration properties {#isprops}
+
+Use this information to configure Content Services to authenticate using Identity Service. Content Services uses Spring Security components because the Keycloak Adapters have been deprecated. Key features and behaviors of the Keycloak Adapter, such as the communication with the Alfresco Identity Service, are preserved. For more information, see the Keycloak documentation [Deprecation of Keycloak adapters](https://www.keycloak.org/2022/02/adapter-deprecation).
+
+Configure the `alfresco-global.properties` file using the below properties:
+
+Alfresco Identity Service Authentication options:
+
+| Property | Description |
+| -------- | ----------- |
+| identity-service.authentication.enabled | Enable or disable authentication via the Identity Service. The default value  is `true`. |
+| identity-service.authentication.validation.failure.silent | Sets whether token validation failure is silent. The default value  is `true`. |
+| identity-service.authentication.defaultAdministratorUserNames | The default administrator user name. The default value  is `admin`. |
+| identity-service.authentication.allowGuestLogin | Sets whether guest logins are allowed. The default value  is `true`. |
+| identity-service.authentication.enable-username-password-authentication | Enable username and login password authentication. The default value  is `true`. |
+
+Specifying the Alfresco Identity Service (Keycloak) details:
+
+| Property | Description |
+| -------- | ----------- |
+| identity-service.realm | Name of the realm configured in the Identity Service. The default value  is `alfresco`. |
+| identity-service.auth-server-url | Base URL of the Identity Service server in the format `https://{server}:{port}/auth`. The default value is `http://localhost:8180/auth`. |
+| identity-service.resource | The **Client ID** for the client created within your realm that points to Content Services. The default value  is `alfresco`. |
+| identity-service.credentials.secret | The **Client Secret** for the client. The default value is an empty string. |
+
+Specifying TLS/mTLS details:
+
+| Property | Description |
+| -------- | ----------- |
+| `identity-service.allow-any-hostname` | If TLS is used this flags allows you to disable host name verification. This might be useful in development environment. The default value is `false`. |

```

Comment:
For consistency, remove the backticks (`) from property names in the first column - affects multiple rows:

```suggestion
| identity-service.allow-any-hostname | If TLS is used, this flag allows you to disable host name verification. This might be useful in a development environment. The default value is `false`. |

```